src/app/wallet/wallet.component.html

```diff
@@ -129,54 +129,44 @@ <h4 class="init-left__title">
+                                                    <ng-container *ngIf="transaction.type === transactionType.CANCEL_INVESTMENT">
+                                                        <b>Cancel investment</b> from
+                                                    </ng-container>
+                                                    <ng-container *ngIf="transaction.type === transactionType.APPROVE_INVESTMENT">
+                                                        <b>Approve investment</b> to
+                                                    </ng-container>
                                                 </div>
                                                 {{ transaction.description }}
                                             </div>
                                             <div class="left__info">
-                                                <span class="info__state-mined" 
-                                                    *ngIf="transaction.state === 'MINED'">
-                                                    <span class="state__invested" 
-                                                            *ngIf="transaction.type === 'INVEST'">
-                                                        1% Invested -
+                                                <span class="info__state-mined"
+                                                      *ngIf="transaction.state === transactionState.MINED">
+                                                    <span class="state__invested"
+                                                          *ngIf="transaction.type === transactionType.INVEST">
+                                                        {{ transaction.share | percent:'1.0-10'}} Invested —
                                                     </span>
                                                 </span>
-                                                <span class="info__state-pending" 
-                                                    *ngIf="transaction.state === 'PENDING'">
+                                                <span class="info__state-pending" *ngIf="transaction.state === 'PENDING'">
                                                     Pending
                                                 </span>
-                                                <span class="info__state-failed" 
-                                                    *ngIf="transaction.state === 'FAILED'">
+                                                <span class="info__state-failed" *ngIf="transaction.state === 'FAILED'">
                                                     Failed
-                                                    <span class="state__invested" 
-                                                            *ngIf="transaction.type === 'INVEST'">
-                                                        1% Invested -
-                                                    </span>
```

Comment:
We have % date on failed investment in design, shold this be removed?

Also, why didn't you use transactionState.PENDING and transactionState.FAILED above?